.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Fixes dotnet/templating#<issue_number>
+
+This PR contains the following changes:
```

Comment:
I really prefer Editor template that has:
## Problem
Write what problem this PR is fixing, or just link issue... Like this:

Fixes https://github.com/dotnet/templating/issues/2805

## Solution

Explain what is fixing, `This PR contains the following changes` feels to long for "placeholder" title.

Reply:
What is also nice about this format, it doesn't "require" issue to open PR... This PR is example of PR without issue...